guides/NEW_APP.md

```diff
@@ -0,0 +1,50 @@
+Creating a new app
+==================
+
+The following steps will guide you through add adding a new app to our system that is built by the CI system and deployed to the live environment.
+
+Prerequisites
+-------------
+
+In order to follow this guide you will need:
+
+* A github account with access to the ONSdigital organisation
+
+* A login for the concourse CI environment
+
+* [dp-cli](https://github.com/ONSdigital/dp-cli)
+
+* Git:
+
+  ```
+  brew install git
+  ```
+
+* Terraform >= 0.12:
+
+  ```
+  brew install terraform
+  ```
+
+* Fly cli (can be downloaded from the links in the corner of the concourse UI)
+
+Getting started
+---------------
+
+These steps will create a new app that is built by CI and deployed to the develop and production environments.  If any of the following github links 404 it is likely that you need to be added to the ONSdigital organisation.
```

Comment:
```suggestion
These steps will create a new app that is built by CI and deployed to the Develop and Production environments.  If any of the following Github links 404 likely you need to be added to the ONSdigital organisation.
```
- 'Github' is a name, capital letter
- Our environments are named 'Production' and 'Develop', capital letter
- "it is likely that you need" -> 'likely you need'; makes the sentence flow easier

Reply:
Our environments are usually referred to as `develop` and `production` (lower case) so I have left the casing as is.